dominion/dominion_ui/src/components/CountryPartners/PartnerContent.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = () => ({
+  root: {
+    flexGrow: 1
+  },
+  title: {
+    paddingBottom: '1rem'
+  }
+});
+
+function PartnerText({ classes, title, description }) {
+  return (
+    <Grid item xs={12} sm={10} md={4} lg={3} xl={3} className={classes.root}>
+      <Typography variant="h4" className={classes.title}>
+        {title}
+      </Typography>
+      <Typography variant="body1">{description}</Typography>
+    </Grid>
```

Comment:
Don't break `Grid` into separate components. This makes the component hard to reusable. This one should just use `div` i.e.
```jsx
<div>
  <Typography ....
    ...
  </Typography>
</div>
```